src/main/java/com/codecool/barbershop/barbershop/booking/BookingController.java

```diff
@@ -46,7 +45,7 @@ public Booking saveNewBooking(@RequestBody BookingReqAddNewBooking bookingReqAdd
 //        Add booking
         Booking newBooking = bookingService.saveNewBooking(bookingReqAddNewBooking);
 //        Send Sms notification
-        smsService.sendSmsNewBooking(newBooking);
+//        smsService.sendSmsNewBooking(newBooking);
```

Comment:
Was it turned off for testing purposes?  
Maybe there could be a configuration setting to turn it off and on. No need for a code change then.